### ISSUE TITLE
Not working with latest glowindex values?

### ISSUE THREAD

**Issue by erwncmps** [iseekwonderful/csgoGlow#301]
I just tried running it today after changing the values in main.ccp with the latest ones posted in the last ticket. No dice. What are you using as of now?

**Comment by Fricker95** [comment]
@erwncmps I will check later.

In the mean time you can try using my fork: [https://github.com/Fricker95/csgoGlow](https://github.com/Fricker95/csgoGlow) and use the -o option, make sure your in insecure mode.

Put the offsets outputted into the console into the Offset.hpp file and see if it works.

No guarantee tho..

**Comment by erwncmps** [comment]
Just did works very well!

—

Erwan Compes
Freelance Designer
+689 89 679 266
Portfolio

**Comment by Fricker95** [comment]
@erwncmps Awesome! could you post the offsets so I can update the fork rn?

**Comment by lutsifer** [comment]
fork deleted

**Comment by Fricker95** [comment]
@lutsifer 

Weird the link does work. The fork is still up though: [https://github.com/Fricker95/csgoGlow](url)

**Comment by Fricker95** [comment]
@lutsifer 

Wired that the link doesn't work.. but the fork is still up and working [https://github.com/Fricker95/csgoGlow](https://github.com/Fricker95/csgoGlow)

**Comment by lutsifer** [comment]
And the wallhack is working just fine <3 <3 <3

**Comment by alextechtai** [comment]
Does this not work right now

**Comment by jakobe15** [comment]
@Fricker95 I seem to be crashing alot when using the new version with the glow on the weapons and entities. I turn off all the features such as -f 1 -t -u but I crash frequently.

**Comment by Fricker95** [comment]
@jakobe15 Ty for the feedback,  I just updated the fork with a more stable version. Lmk if its still crashing

**Comment by jakobe15** [comment]
@Fricker95 Yeah, the crashes are less frequent but it seems like it mostly happens when starting another match or loading into another map.

**Comment by Fricker95** [comment]
@jakobe15 Ok i will look into it, Im also fixing the colors for the entity types enums. When your loading are you on secure servers? Comp of casual?

**Comment by jakobe15** [comment]
@Fricker95 Yeah, it’s whenever I load into another Broken Fang Comp Match.

**Comment by Fricker95** [comment]
@jakobe15 I updated the fork, I think its fixed. lmk

**Comment by jakobe15** [comment]
@Fricker95 Sadly, I just crashed in a game, I ended up going back to the older fork which works fine but I feel like your new fork feels much much smoother.

**Comment by Fricker95** [comment]
@jakobe15 Yes keep using the older version while this one is getting fixed!

I updated the fork again and I haven't experienced a crash (on non-secure servers) I believe it was the 
```
"25C_BaseCSGrenadeProjectile",
"17C_DecoyProjectile",
"19C_MolotovProjectile",
"25C_SensorGrenadeProjectile",
"24C_SmokeGrenadeProjectile",
```
that were the issue, because when they explode the entity is deallocated and with the read/write timings it was probably writing after it had been deallocated..
So I disabled glowing grenades when you throw them.

As for the smoothness Its because the default refresh rate is now 10000 instead of 1000

**Comment by jakobe15** [comment]
@Fricker95 Alright, I’ll do just that. Just wondering the command -r 10000 would work to change the refresh rate when executing right?

**Comment by Fricker95** [comment]
@jakobe15 
Yes, with: `sudo ./Wall -r 10000` you don't need to recompile to change the refresh rate.

But with the old fork with a high refresh rate it was very "flickery" and kinda laggy.

With the new fork if you have a powerful cpu you can even try `sudo ./Wall -r 0` reduces the flickering. 

I will also remove the glow for weapons that are equipped, so players don't flicker from colored and white glow..

**Comment by jakobe15** [comment]
@Fricker95 Yep, adding the -r 10000 line made it so much smoother! Thank you for the help.

**Comment by Fricker95** [comment]
@jakobe15 
np! :) I updated it again it should crash less...
